.pre-commit-config.yaml

```diff
@@ -945,7 +957,10 @@ repos:
         entry: ./scripts/ci/pre_commit/check_system_tests.py
         language: python
         files: ^tests/system/.*/example_[^/]*\.py$
-        exclude: ^tests/system/providers/google/cloud/bigquery/example_bigquery_queries\.py$
+        exclude: >
+          (?x)
+          ^tests/system/providers/google/cloud/bigquery/example_bigquery_queries\.py$ |
+          ^tests/system/providers/standard/time/operators/example_branch_datetime_operator.py
```

Comment:
Why does this need an exclusion? This is not a system test. Does the check raise a false-positive and need a fix?

Reply:
Good catch thank you, have removed that from exclusions.